lms/urls.py

```diff
@@ -81,6 +78,24 @@
 
 )
 
+if settings.FEATURES["ENABLE_COMBINED_LOGIN_REGISTRATION"]:
+    # Backwards compatibility with old URL structure, but serve the new views
+    urlpatterns += (
+        url(r'^login$', 'student_account.views.login_and_registration_form',
+            {'initial_mode': 'login'}, name="signin_user"),
+        url(r'^register$', 'student_account.views.login_and_registration_form',
+            {'initial_mode': 'register'}, name="register_user"),
```

Comment:
They will continue to be served by the `login_and_registration_form` view, alongside `/login` and `/register`. See [student_account/urls.py](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/student_account/urls.py). I've created [ECOM-1017](https://openedx.atlassian.net/browse/ECOM-1017) to track their eventual removal.